test/integration/run-command.js

```diff
@@ -30,11 +33,49 @@ function runCommand(path, args, options) {
         return;
       }
       if (code !== 0) {
-        reject(new Error('exit code: ' + code));
+        reject(new Error(
+          `Error running ${path} with args ${args}. Got exit code: ${code}`));
         return;
       }
       resolve();
     });
+  }).catch((err) => {
+    // If the command was successful there's no need to print anything to the
+    // console, but if it failed then its output is probably helpful.
+    // Print out the output, then reject the final result with the original
+    // error.
+    return contents.then((out) => {
+      console.log(`Output of failed command 'node ${path} ${args.join(' ')}' in directory ${options.cwd}`);
+      console.log(out);
```

Comment:
Yes, I've confirmed that the output is successfully sent out. See e.g. https://travis-ci.org/Polymer/polymer-cli/jobs/202104841#L829